cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -227,16 +229,21 @@ class AceCR(cirq.Gate):
 
     The typical AceCR in literature is a positive half-CR, then X on "Z side", then negative
     half-CR ("Z side" and "X side" refer to the two sides of the underlying ZX interactions).
-
-    Args:
-        rads: Angle of rotation for CR gate (i.e., twice the angle for each echoed half-CR).
-        sandwich_rx_rads: Angle of rotation for an rx gate applied to the "X side" simultaneously
-            with the X gate on the "Z side".
     """
 
     def __init__(
         self, rads: Union[str, cirq.TParamVal] = np.pi / 2, sandwich_rx_rads: cirq.TParamVal = 0
     ) -> None:
+        """Initializes an AceCR gate.
+
+        Args:
+            rads: Angle of rotation for CR gate (i.e., twice the angle for each echoed half-CR).
+            sandwich_rx_rads: Angle of rotation for an rx gate applied to the "X side"
+                simultaneously with the X gate on the "Z side".
+
+        Raises:
+            ValueError: If the polarity of `rads` is not '+-' or '-+'.
```

Comment:
```suggestion
            ValueError: If the polarity of `rads` is a string other than '+-' or '-+'.
```
(non-string values are ok)

Reply:
Corrected.